src/main/scala/org/broadinstitute/dsde/agora/server/ServerInitializer.scala

```diff
@@ -32,8 +36,18 @@ class ServerInitializer(val config: Config) extends LazyLogging {
   }
 
   private def startWebServiceActors() {
+    implicit val timeout = Timeout(5.seconds)
```

Comment:
Needs to be an implicit. Going to keep it scoped close to the actual ask so it doesn't get passed around places we don't expect